src/Tickets/Seating/app/blockEditor/store/selectors.js

```diff
@@ -37,15 +37,27 @@ export const selectors = {
 		});
 	},
 	getCurrentLayoutId(state) {
+		if (!!!state.isUsingAssignedSeating) {
```

Comment:
Here and elsewhere why not use just one `!`? `!!!` is `negation of negation of negation of state.isUsingAssignedSeating` that results in `negation of state.isUsingAssignedSeating`. Am I not getting something?

Reply:
No you are right. I was not thinking it seems. Fixed